src/xcode/ENA/ENA/Source/Scenes/ContactDiary/__tests__/ENAUITests_07_ContactJournalUITests.swift

```diff
@@ -280,17 +280,148 @@ class ENAUITests_07_ContactJournalUITests: XCTestCase {
 		// check count for day entries: 1 add entry cell
 		XCTAssertEqual(app.descendants(matching: .table).firstMatch.cells.count, 1)
 
-		addLocationToDayEntry("Pommesbude")
+		addLocationToDayEntry("Pommesbude", phoneNumber: "12345678", eMail: "pommes@bude.de")
 
 		// check count for day entries: 1 add entry cell + 1 location added
 		XCTAssertEqual(app.descendants(matching: .table).firstMatch.cells.count, 2)
 
-		addLocationToDayEntry("Supermarkt")
+		addLocationToDayEntry("Supermarkt", phoneNumber: "12345678", eMail: "super@markt.de")
 
 		// check count for day entries: 1 add entry cell + 2 locations added
 		XCTAssertEqual(app.descendants(matching: .table).firstMatch.cells.count, 3)
 	}
 
+	func testDetailsSelectionOfPersonEncounter() {
+		app.launchArguments.append(contentsOf: ["-diaryInfoScreenShown", "YES"])
+
+		navigateToJournalOverview()
+
+		// Select 3th cell.
+
+		XCTAssertTrue(app.descendants(matching: .table).firstMatch.cells.element(boundBy: 3).waitForExistence(timeout: .medium))
+		app.descendants(matching: .table).firstMatch.cells.element(boundBy: 3).tap()
+
+		// Add person.
+
+		addPersonToDayEntry("Marcus Mustermann")
+
+		// Select details of encounter.
+
+		XCTAssertTrue(app.segmentedControls[AccessibilityIdentifiers.ContactDiaryInformation.Day.maskSituationSegmentedControl].firstMatch.buttons.element(boundBy: 1).waitForExistence(timeout: .medium))
+		app.segmentedControls[AccessibilityIdentifiers.ContactDiaryInformation.Day.maskSituationSegmentedControl].firstMatch.buttons.element(boundBy: 1).tap()
+
+		XCTAssertTrue(app.segmentedControls[AccessibilityIdentifiers.ContactDiaryInformation.Day.settingSegmentedControl].firstMatch.buttons.element(boundBy: 1).waitForExistence(timeout: .medium))
+		app.segmentedControls[AccessibilityIdentifiers.ContactDiaryInformation.Day.settingSegmentedControl].firstMatch.buttons.element(boundBy: 1).tap()
```

Comment:
Improved readbility:
```suggestion
		let maskSituationButton = app.segmentedControls[AccessibilityIdentifiers.ContactDiaryInformation.Day.maskSituationSegmentedControl].firstMatch.buttons.element(boundBy: 1)
		XCTAssertTrue(maskSituationButton.waitForExistence(timeout: .medium))
		maskSituationButton.tap()

		let settingButton = app.segmentedControls[AccessibilityIdentifiers.ContactDiaryInformation.Day.settingSegmentedControl].firstMatch.buttons.element(boundBy: 1)
 		XCTAssertTrue(settingButton.waitForExistence(timeout: .medium))
		settingButton.tap()
```